_summaries/qi2016pointnet_1.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: summary
+title: Summary
+paper: qi2016pointnet
+author: slycane9
+score: 9
+---
+* What is the core idea?
+
+Before this paper, state-of-the-art 3D classification/segmentation models all required 3D input to be transformed into voxel grids or multiple image views.  However, this paper introduces PointNets, which only requires point clouds as input to its model.  Using point clouds as input avoids issues with dense image inputs, and avoid irregularities that are difficult to learn from.
+
+* How is it realized (technically)?
+
+![arch](qi2016pointnet_1a.png)
+
+Above is the architecture for PointNet.  The first layers go through an input and feature transformer.  These transformers are themselves mini networks and protect against variance in the point cloud input (ensuring we regard different transformations of the point cloud the same way).
```

Comment:
I would to into more detail about what the mini networks are (also mention that they are the T-nets in the image). I would also discuss the regularization that is done to the feature transformation matrices, and why it is done.